galaxy_ng/app/migrations/0028_move_perms_to_roles.py

```diff
@@ -0,0 +1,399 @@
+        ("galaxy", "change_containerregistryremote"),
+        ("galaxy", "delete_containerregistryremote"),
+    ), "galaxy.content_admin"),
+
+    # COLLECTIONS
+    ((
+        ("galaxy", "add_namespace"),
+        ("galaxy", "change_namespace"),
+        ("galaxy", "delete_namespace"),
+        ("galaxy", "upload_to_namespace"),
+        ("ansible", "delete_collection"),
+        ("ansible", "change_collectionremote"),
+        ("ansible", "view_collectionremote"),
+        ("ansible", "modify_ansible_repo_content"),
+    ), "galaxy.collection_admin"),
+    ((
+    ("galaxy", "add_namespace"),
+    ("galaxy", "change_namespace"),
+    ("galaxy", "upload_to_namespace"),
+    ), "galaxy.collection_publisher"),
+    ((
+        ("ansible", "change_collectionremote"),
+        ("ansible", "view_collectionremote"),
+        ("ansible", "modify_ansible_repo_content"),
+    ), "galaxy.collection_curator"),
+    ((
+        ("galaxy", "change_namespace"),
+        ("galaxy", "upload_to_namespace"),
+    ), "galaxy.collection_namespace_owner"),
+
+    # EXECUTION ENVIRONMENTS
+    ((
+        ("container", "delete_containerrepository"),
+        ("container", "namespace_change_containerdistribution"),
+        ("container", "namespace_modify_content_containerpushrepository"),
+        ("container", "namespace_push_containerdistribution"),
+        ("container", "add_containernamespace"),
+        ("container", "change_containernamespace"),
+        # Excluding this because it's only used for object assignment, not model
+        # ("container", "namespace_add_containerdistribution"),
+        ("galaxy", "add_containerregistryremote"),
+        ("galaxy", "change_containerregistryremote"),
+        ("galaxy", "delete_containerregistryremote"),
+    ), "galaxy.execution_environment_admin"),
+    ((
+        ("container", "namespace_change_containerdistribution"),
+        ("container", "namespace_modify_content_containerpushrepository"),
+        ("container", "namespace_push_containerdistribution"),
+        ("container", "add_containernamespace"),
+        ("container", "change_containernamespace"),
+        # ("container", "namespace_add_containerdistribution"),
+    ), "galaxy.execution_environment_publisher"),
+    ((
+        ("container", "change_containernamespace"),
+        ("container", "namespace_push_containerdistribution"),
+        ("container", "namespace_change_containerdistribution"),
+        ("container", "namespace_modify_content_containerpushrepository"),
+        # ("container", "namespace_add_containerdistribution"),
+    ), "galaxy.execution_environment_namespace_owner"),
+    ((
+        ("container", "namespace_push_containerdistribution"),
+        ("container", "namespace_change_containerdistribution"),
+        ("container", "namespace_modify_content_containerpushrepository"),
+    ), "galaxy.execution_environment_collaborator"),
+    ((
+        ("ansible", "modify_ansible_repo_content"),
+    ), "galaxy.content_admin"),
```

Comment:
should this be removed? `galaxy.content_admin` is already in `GLOBAL_PERMISSION_TRANSLATOR`

Reply:
oh, good catch